pkg/manifests/manifests.go

```diff
@@ -138,6 +138,7 @@ var (
 	NodeExporterMinimalServiceMonitor      = "node-exporter/minimal-service-monitor.yaml"
 	NodeExporterPrometheusRule             = "node-exporter/prometheus-rule.yaml"
 	NodeExporterKubeRbacProxySecret        = "node-exporter/kube-rbac-proxy-secret.yaml"
+	NodeExporterConfigMap                  = "node-exporter/accelerators-configmap.yaml"
```

Comment:
```suggestion
	NodeExporterAcceleratorsConfigMap                  = "node-exporter/accelerators-configmap.yaml"
```
Maybe a more descriptive name?

Reply:
Fixed